app/Http/Controllers/WikiController.php

```
@@ -45,39 +47,45 @@ public function index(Request $request)
+        if (!$isAdmin) {
+            $wikiQuery->approved();
+        }
+
+        if ($query !== null && $query !== '') {
+            $wikiQuery->whereTranslationLike('title', '%'.$query.'%');
         }
 
+        $wikidata = $wikiQuery->orderBy('created_at', 'desc')->paginate($perPage);
+
         $total = $wikidata->total();
         $offset = ($page - 1) * $perPage;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$offset'. (undefined)

(UnusedLocalVariable)